trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/record-correction-helpers.ts

```diff
@@ -30,7 +30,7 @@ export const mapToRecordCorrectionTableRowViewModel = (
  */
 export const mapToRecordCorrectionViewModel = (recordCorrectionDetails: FeeRecordCorrectionSummary[]): RecordCorrectionsViewModel => {
   const mappedRows = recordCorrectionDetails.map((recordCorrection) => {
-    const displayStatus = getFeeRecordDisplayStatus(recordCorrection.status);
+    const displayStatus = setRecordCorrectionDisplayStatus(recordCorrection.isCompleted);
```

Comment:
Why are we determining display status here, but then determining status inside `mapToRecordCorrectionTableRowViewModel`? Feels a bit inconsistent. I think this call should happen inside that function too.

Reply:
this is following what is being done in other helpers but will change it here

Reply:
I think if this is done in other helpers it is because there is a group that all use the same status, so it is more efficient to calculate upfront once, or at least that should be the reason it is done like that in other places